movieApp/asset/app.js

```diff
@@ -0,0 +1,50 @@
+//const API_KEY=daf7b11fd7dfe1a364da7a86d79ba727;
+const IMG_URL="https://image.tmdb.org/t/p/w500";
+const url="https://api.themoviedb.org/3/search/movie?api_key=daf7b11fd7dfe1a364da7a86d79ba727";
+
+//selecting search elements from DOM
+const inputElement=document.querySelector("#searchBox");
+const movieSearchable=document.querySelector("#movies-searchable");
+
+inputElement.onsearch=function(event){//searchFunction()
+    
+    //event.preventDefault();
+    const value=inputElement.value;
+    const newUrl= url+'&query='+value;
```

Comment:
use string literals: 
```js
const newUrl = `${url}&query=${value}`;
```

and you don't need this variable actually. You can do this operation directly in fetch:
```js
fetch(`${url}&query=${value}`)...
```